packages/admin/admin/src/mui/menu/ItemGroup.tsx

```diff
@@ -2,51 +2,61 @@ import { Box, ComponentsOverrides, Theme, Tooltip, Typography } from "@mui/mater
+import { MenuItemProps } from "./Item";
+import { MenuItemRouterLinkProps } from "./ItemRouterLink";
+
 export type MenuItemGroupClassKey = "root" | "title" | "titleContainer";
 
 const styles = (theme: Theme) =>
     createStyles<MenuItemGroupClassKey, MenuItemGroupProps>({
         root: { marginTop: theme.spacing(8) },
         title: {
             fontWeight: theme.typography.fontWeightBold,
-            fontSize: ({ drawerOpen }) => (drawerOpen ? 14 : 12),
-            border: ({ drawerOpen }) => (drawerOpen ? `2px solid ${theme.palette.common.white}` : `2px solid ${theme.palette.grey[100]}`),
-            borderRadius: ({ drawerOpen }) => (drawerOpen ? "initial" : 20),
-            padding: ({ drawerOpen }) => (drawerOpen ? "0" : theme.spacing(2, 4)),
+            fontSize: ({ isMenuOpen }) => (isMenuOpen ? 14 : 12),
+            border: ({ isMenuOpen }) => (isMenuOpen ? `2px solid ${theme.palette.common.white}` : `2px solid ${theme.palette.grey[100]}`),
+            borderRadius: ({ isMenuOpen }) => (isMenuOpen ? "initial" : 20),
+            padding: ({ isMenuOpen }) => (isMenuOpen ? "0" : theme.spacing(0, 1.5)),
             lineHeight: "20px",
-            color: ({ drawerOpen }) => (drawerOpen ? `${theme.palette.common.black}` : `${theme.palette.grey[300]}`),
+            color: ({ isMenuOpen }) => (isMenuOpen ? `${theme.palette.common.black}` : `${theme.palette.grey[300]}`),
         },
         titleContainer: {
             borderBottom: `1px solid ${theme.palette.grey[50]}`,
             display: "flex",
-            justifyContent: ({ drawerOpen }) => (drawerOpen ? "flex-start" : "center"),
-            padding: ({ drawerOpen }) => `${theme.spacing(2)} ${drawerOpen ? theme.spacing(4) : 0}`,
+            justifyContent: ({ isMenuOpen }) => (isMenuOpen ? "flex-start" : "center"),
+            padding: ({ isMenuOpen }) => `${theme.spacing(2)} ${isMenuOpen ? theme.spacing(4) : 0}`,
         },
     });
 
 export interface MenuItemGroupProps {
     title: string;
-    drawerOpen?: boolean;
+    isMenuOpen?: boolean;
 }
 
-const ItemGroup: React.FC<React.PropsWithChildren<WithStyles<typeof styles> & MenuItemGroupProps>> = ({ title, children, classes, drawerOpen }) => {
+const ItemGroup: React.FC<React.PropsWithChildren<WithStyles<typeof styles> & MenuItemGroupProps>> = ({ title, children, classes, isMenuOpen }) => {
     const initialTitle = title;
     function getInitials(title: string) {
         const words = title.split(/\s+/).filter((word) => word.match(/[A-Za-z]/));
         return words.map((word) => word[0].toUpperCase()).join("");
     }
 
-    if (drawerOpen === false) {
+    if (isMenuOpen === false) {
         title = getInitials(title);
     }
 
+    const childElements = React.Children.map(children, (child: MenuChild) => {
+        return React.cloneElement<MenuCollapsibleItemProps | MenuItemRouterLinkProps | MenuItemProps>(child, {
+            isMenuOpen,
+        });
+    });
```

Comment:
With the menu context it shouldn't be necessary to clone the children

Reply:
Actually I can't really get rid of it because i need this prop for styling. It would create a big overhead.